pkg/cmd/cli/cmd/process.go

```diff
@@ -58,5 +100,34 @@ Examples:
 		},
 	}
 	cmd.Flags().StringP("filename", "f", "", "Filename or URL to file to use to update the resource")
+	cmd.Flags().StringP("value", "v", "", "Set value for the specified parameter")
+	cmd.Flags().BoolP("print", "p", false, "Do not process but only print available parameters")
+	cmd.Flags().Bool("no-headers", false, "When using the default output, don't print headers")
 	return cmd
 }
+
+// parseMultiFlagValue parses multiple values from the command-line arguments if
+// they are repeated more then once. This helper supports both long and
+// short-hand flags.
+func parseMultiFlagValue(f *pflag.Flag) []string {
```

Comment:
I know that this looks like a dirty hack, however I was not able to find obvious way to collect values from multiple arguments (I know that `docker` does this with `-e`)... This is based on cobra code, so probably safe for now...